### ISSUE TITLE
Provide Obvious Contact Options to report a11y Issues

### ISSUE THREAD

**Issue by tbsbdr** [opencloud-eu/web#737]
### Initial Checklist

- [x] I understand this is a user story and questions should be posted in the [Community Discussions](https://github.com/orgs/opencloud-eu/discussions/)
- [x] I searched [issues](https://github.com/opencloud-eu/web/issues?q=is%3Aissue) and couldn’t find anything (or linked relevant results below)

### User Story

As the vendor of OpenCloud I want to know if users encounter a11y issues so that we can fix them.

### User Value

- Anyone can use OpenCloud

### Acceptance Criteria

- Provide an obvious and easy to find way to report a11y issues
- Extreme low-threshold ("niederschwellig"): There should be as few as possible barrieres to report a11y issues, eg. gh account-obligation is a no-go as prerequisite to report an a11y issue.

**Comment by db-ot** [comment]
Dear @tbsbdr we should definitly here create a page called 
"Accessibility Statement" ( also done in other projects )

This Accessibility Statement outlines which parts of OpenCloud are compliant with accessibility (A11Y) standards, which areas currently have known issues, and what those issues are.

Our goal is to make our software accessible to everyone, including users with disabilities. We are continuously working to identify and fix accessibility barriers.

At the end of this page, you will find contact information or a contact form that you can use to report any accessibility problems, suggest improvements, or request assistance.

**Comment by tbsbdr** [comment]
@db-ot I consider the a11y statement as bare-minimum to acquire the certification.
 - mail is okay and one possible option to report an issue but still has an account obligation (mail account)
 - speaking of contact forms: how about in (addition to the a11y statement) creating a form in lime survey where you can just drop your a11y finding?
 - any idea where we could place the link to the a11y statement in the ui?

**Comment by db-ot** [comment]
@tbsbdr perfect, me too!

- mail: just an alias mail adress ( a11y@ or accessibility@ ), like we last time talked about for other teams or topic 
- limesurvey: perfect idea, i guess this i also can create
- attention: we have here 2 touchpoints for a11y statement ( marketing website: opencloud.eu - which is a seperate stack from our software and opencloud - as software application ) - i would suggest on opencloud.eu in the footer ( close by imprint or into the "about us" que ) in the software application under documentation footer: https://demo.opencloud.eu/external-sites/documentation ( also think about to have legal links on the documentation page? )

**Comment by AlexAndBear** [comment]
I don't know limesurvey, sounds more like a survey platform? Google forms could also be an alternative

**Comment by tbsbdr** [comment]
## Version 1

> Report a11y Issues
> 
> Dear Customer, To ensure that our digital services are accessible to everyone, your feedback is especially important to us.
> If you have encountered any barriers or other difficulties while using our services, please let us know using this form.
> Your OpenCloud Team
> 
> Please describe any barriers or difficulties you have encountered.
> Please also let us know which browser, operating system, and device (e.g. computer, smartphone, or tablet) you were using.
> 
> Submit

## Version 2

# Report Accessibility Barriers

To help ensure that **OpenCloud** is accessible to everyone, you can report any accessibility barriers using this form.  
Please use this form **only** to report accessibility issues related to the **OpenCloud software** or the website **[www.opencloud.eu](https://www.opencloud.eu)**.  
For all other inquiries, please use our general [contact form](https://opencloud.eu/de/kontakt).

We treat every accessibility barrier as a bug and track it publicly on [GitHub](https://github.com/search?q=label%3AType%3AAccessibility-Defect&type=issues).

**– Your OpenCloud Team**

---

## Please include the following information:

- A description of the barrier or difficulty you encountered  
- The **browser**, **operating system**, and **device** you were using  
  _(e.g. Chrome on Windows 11, iPhone with iOS 17, Android tablet, etc.)_

---

[**Submit**]
